src/ImageSharp.Web.Providers.AWS/Caching/AWSS3StorageCache.cs

```diff
@@ -119,5 +122,53 @@ public Task SetAsync(string key, Stream stream, ImageCacheMetadata metadata)
 
             return null;
         }
+
+        /// <summary>
+        /// <see href="https://github.com/aspnet/AspNetIdentity/blob/b7826741279450c58b230ece98bd04b4815beabf/src/Microsoft.AspNet.Identity.Core/AsyncHelper.cs"/>
+        /// </summary>
+        internal static class AsyncHelper
```

Comment:
This should probably be made into a private class or otherwise extracted into its own file:
```suggestion
        private static class AsyncHelper
```